doc/cm/cm_base.yml

```diff
@@ -1738,3 +1738,11 @@
          Maximum value of RLIMIT_MEMLOCK. Note: if less than current value
          is set, current value is adjusted too.
          Name: empty
+
+    - oid: "/agent/ta_events"
```

Comment:
IMHO it is weird. RCF should care about it itself. It restarts the agent and it may subscribe to events using RCF protocol just after restart. Usage of CS here just makes the picture more and more complicated.
If you which to expose the information (make it clear which events are subscribed to), you may do it (read-only volatile).